### ISSUE TITLE
Restring crashes on a few Huawei phones when highlighting TextView

### ISSUE THREAD

**Issue by edgar-zigis** [B3nedikt/restring#51]
Hi again, so after updating to latest version I got this issue which is quite massive:

```
Fatal Exception: java.lang.NullPointerException: Parameter specified as non-null is null: method s.y.d.m.f, parameter defType
       at dev.b3nedikt.restring.internal.RestringResources.getIdentifier(RestringResources.java:7)
       at com.android.internal.widget.FloatingToolbar.getThemeId(FloatingToolbar.java:1931)
       at com.android.internal.widget.FloatingToolbar.applyDefaultTheme(FloatingToolbar.java:1912)
       at com.android.internal.widget.FloatingToolbar.<init>(FloatingToolbar.java:158)
       at com.android.internal.policy.HwPhoneWindow.getFloatingToolbar(HwPhoneWindow.java:282)
       at com.android.internal.policy.DecorView.createFloatingActionMode(DecorView.java:2055)
       at com.android.internal.policy.DecorView.createActionMode(DecorView.java:1898)
       at com.android.internal.policy.DecorView.startActionMode(DecorView.java:1027)
       at com.android.internal.policy.DecorView.startActionModeForChild(DecorView.java:984)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:934)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:934)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:934)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:934)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:934)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:934)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:934)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:934)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:934)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:934)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:934)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:934)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:934)
       at android.view.View.startActionMode(View.java:6531)
       at android.widget.Editor.startInsertionActionMode(Editor.java:2072)
       at android.widget.TextView.onTouchEvent(TextView.java:9950)
```
Most popular devices having this issue:
Huawei P30 Pro (Lite too), Y7 Prime 2019, P20 Pro (Lite too), Honor 8X.
Also there are instances of Samsung A9 (2018) and OnePlus 6T.

At the moment we can't use this library, because there are many crash instances.

**Comment by edgar-zigis** [comment]
Another instance of this here:

```
Fatal Exception: java.lang.NullPointerException: Parameter specified as non-null is null: method s.y.d.m.f, parameter defType
       at dev.b3nedikt.restring.internal.RestringResources.getIdentifier(RestringResources.java:35)
       at com.android.internal.widget.FloatingToolbar.getThemeId(FloatingToolbar.java:1841)
       at com.android.internal.widget.FloatingToolbar.applyDefaultTheme(FloatingToolbar.java:1822)
       at com.android.internal.widget.FloatingToolbar.<init>(FloatingToolbar.java:149)
       at com.android.internal.policy.HwPhoneWindow.getFloatingToolbar(HwPhoneWindow.java:241)
       at com.android.internal.policy.DecorView.setHandledFloatingActionMode(DecorView.java:1809)
       at com.android.internal.policy.DecorView.setHandledActionMode(DecorView.java:1645)
       at com.android.internal.policy.DecorView.startActionMode(DecorView.java:909)
       at com.android.internal.policy.DecorView.startActionModeForChild(DecorView.java:864)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:827)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:827)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:827)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:827)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:827)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:827)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:827)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:827)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:827)
       at android.view.ViewGroup.startActionModeForChild(ViewGroup.java:827)
       at android.view.View.startActionMode(View.java:5840)
       at android.widget.Editor.startInsertionActionMode(Editor.java:1971)
       at android.widget.TextView.onTouchEvent(TextView.java:8810)
```

**Comment by B3nedikt** [comment]
@edgar-zigis Fixed in Restring 5.1.0

**Comment by edgar-zigis** [comment]
Will check this out again.
